cpp/ql/src/Security/CWE/CWE-022/TaintedPath.ql

```diff
@@ -88,6 +88,11 @@ module TaintedPathConfig implements DataFlow::ConfigSig {
       hasUpperBoundsCheck(checkedVar)
     )
   }
+
+  predicate isBarrierOut(DataFlow::Node node) {
+    // make sinks barriers so that we only report the closest instance
+    isSink(node)
```

Comment:
This only makes the sink a barrier (out), but the PR description intended you also wanted to make the source a barrier (in). Is that intentional?

I don't think making `FlowSource` an in barrier will do anything, so I'm quite happy with this as-is, but I just wanted to point out the discrepancy between the PR description and the code 👍

Reply:
Ah, well spotted - I tried both, it didn't make any difference in practice so I dropped the sources bit.  Looks like I forgot that when I wrote the PR description though!